framework/src/imgui_manager.cpp

```diff
@@ -392,4 +413,17 @@ namespace gvk
 		avk::assign_and_lifetime_handle_previous(mRenderpass, std::move(newRenderpass), lifetimeHandlerLambda);
 		avk::assign_and_lifetime_handle_previous(mClearRenderpass, std::move(newClearRenderpass), lifetimeHandlerLambda);
 	}
+
+	ImTextureID imgui_manager::get_or_create_texture(avk::image_sampler& aImageSampler)
+	{
+		std::vector<avk::descriptor_set> sets = mImTextureDescriptorCache.get_or_create_descriptor_sets({
+			avk::descriptor_binding(0, 0, aImageSampler, avk::shader_type::fragment)
```

Comment:
```suggestion
			avk::descriptor_binding<avk::image_sampler_t>(0u, 0u, 1u, avk::shader_type::fragment)
```
Maybe that ^ suffices.

Reply:
Just ignore this ;)